docs/blockchain_node-reference.md

```diff
@@ -193,18 +193,20 @@ Get account details for a given account address.
-| ---------------------------- | ------ | ----------- | ---------------------------------------------------------------------------------------------------- |
-| result                       | object |             | Account                                                                                              |
-| result.address               | string |             | Address of the account                                                                               |
-| result.balance               | number |             | HNT balance of the account in bones                                                                  |
-| result.nonce                 | number |             | The current nonce for the account                                                                    |
-| result.speculative_nonce     | number |             | The larger of the maximum pending balance nonce or the current nonce.                                |
-| result.dc_balance            | number |             | Data credit balance of the account                                                                   |
-| result.dc_nonce              | number |             | The current data credit nonce for the account                                                        |
-| result.sec_balance           | number |             | Security token balance of the account                                                                |
-| result.sec_nonce             | number |             | The current security token nonce for the account                                                     |
-| result.sec_speculative_nonce | number |             | The larger of the maximum pending security nonce or the current security token nonce for the account |
+| Name                          | Type   | Constraints | Description                                                                                                       |
+| ----------------------------- | ------ | ----------- | ----------------------------------------------------------------------------------------------------------------- |
+| result                        | object |             | Account                                                                                                           |
+| result.address                | string |             | Address of the account                                                                                            |
+| result.balance                | number |             | HNT balance of the account in bones                                                                               |
+| result.mobile_balance         | number |             | MOBILE balance of the account in bones                                                                            |
+| result.iot_balance            | number |             | IOT balance of the account in bones                                                                               |
```

Comment:
will this ever actually work since we're currently planning _not_ to activate the iot subnetwork on the current L1 chain?